walkthrough/src/modules/app/walkthrough/_javascript.jade

```diff
@@ -142,7 +142,7 @@ questions.Questions--2
       answer-toggle(correct="false")
         :markdown
           ```
-          .SearchForm .isSuperUser.Btn {
+          .SearchForm .isSuperUser .Btn {
```

Comment:
sounds good. everything looks good in the PR except for [this change](https://github.com/gilbox/css-bliss/pull/7#discussion_r28845942)